pkg/volume/csi/csi_test.go

```diff
@@ -244,8 +220,6 @@ func TestCSI_VolumeAll(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.CSIInlineVolume, !test.disableFSGroupPolicyFeatureGate)()
```

Comment:
do you know why CSIInlineVolume was tied to fsgroup policy feature gate?

Reply:
There was just the one test case [added here](https://github.com/kubernetes/kubernetes/commit/4d2d063635c86aceb626d88c1d785e0463109df8#diff-805a29cf10e2aafaae4e1665d310fa4edd041e32da71bba9cf06f7200368cd70R96-R114) to test `supportsFSGroup` with the feature gate disabled because of this check:
https://github.com/kubernetes/kubernetes/blob/96a32be18dfbd38ff5cb1df4d68528e2cf226958/pkg/volume/csi/csi_mounter.go#L417-L421
In other words, the current behavior (without my PR) is to fail setting FSGroup on CSI Inline Volumes when the feature gate is disabled. TBH, I don't think we could actually get that far if the feature gate is disabled since the volume couldn't be created in the first place.

Reply:
Ah, and to be clear I think `disableFSGroupPolicyFeatureGate` was a misleading variable name since it only ever controlled the `CSIInlineVolume` feature gate.